core/test/unit/services/auth/api-key/admin_spec.js

```diff
@@ -0,0 +1,209 @@
+const {authenticateAdminApiKey} = require('../../../../../server/services/auth/api-key/admin');
+const common = require('../../../../../server/lib/common');
+const jwt = require('jsonwebtoken');
+const models = require('../../../../../server/models');
+const should = require('should');
+const sinon = require('sinon');
+const testUtils = require('../../../../utils');
+
+const sandbox = sinon.sandbox.create();
+
+describe('Admin API Key Auth', function () {
```

Comment:
@kirrg001 this test should also be disabled for now and reenabled when `auth.authenticate.authenticateAdminApiKey` middleware lands

Reply:
actually, this seems to just test the middleware itself, not the actual endpoint, so safe to go in as is